data/agama_auto/sle_default_staging.jsonnet

```diff
@@ -23,13 +25,18 @@
   },
   software: {
     packages: [],
-    extraRepositories: [
-      {
-        alias: 'TEST_0',
-        url: '{{INCIDENT_REPO}}',
-        allowUnsigned: true
-      }
-    ],
+    extraRepositories:
+      if repo != "" then
```

Comment:
For jsonnet, please use '' instead of "".

Reply:
Thanks much!